sites/footers/footers-en.html

```diff
@@ -1,48 +1,195 @@
 ---
 {
-	"title": "Content page including complete footer",
+	"title": "Footer",
 	"language": "en",
 	"altLangPage": "footers-fr.html",
 	"secondlevel": false,
-	"dateModified": "2022-05-18",
-	"share": "true"
+	"dateModified": "2022-06-20",
+	"share": "true",
+	"contextualFooter": {
+		"title": "Page-specific contextual footer title",
```

Comment:
I'd keep the wording like so (with square brackets to show that it is configurable / a placeholder):
```suggestion
		"title": "[Contextual footer title]",
```
And square brackets should consequently be added in every example where such title is used.

Observation: This is subject to change to align with DTO documentation later on.